tools/NuGet/nuspec/DynamoVisualProgramming.Core.nuspec

```diff
@@ -1,26 +1,25 @@
 <?xml version="1.0"?>
-<package >
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
   <metadata>
     <id>DynamoVisualProgramming.Core</id>
     <version>2.0.0-beta1</version>
     <authors>Autodesk</authors>
     <owners>Autodesk</owners>
     <licenseUrl>http://opensource.org/licenses/MIT</licenseUrl>
     <projectUrl>https://github.com/DynamoDS/Dynamo</projectUrl>
+    <iconUrl>https://raw.githubusercontent.com/DynamoDS/Dynamo/master/doc/distrib/Images/logo_square_32x32.png</iconUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>Unit and system test infrastructure for Dynamo. It contains the following files:
-		1)DynamoCore.dll
-		2)ProtoCore.dll
-		3)DynamoShapeManager.dll 
-		4)DynamoUtilities.dll
-	</description>
+1) DynamoCore.dll
+2) ProtoCore.dll
+3) DynamoShapeManager.dll 
+4) DynamoUtilities.dll</description>
     <summary>This package contains the core assemblies for Dynamo.</summary>
     <copyright>Copyright Autodesk 2016</copyright>
-    <iconUrl>https://raw.githubusercontent.com/DynamoDS/Dynamo/master/doc/distrib/Images/logo_square_32x32.png</iconUrl>
     <dependencies>
-        <dependency id="Prism" version="4.1"/>
-        <dependency id="NUnit" version="2.6.3"/>
-		<dependency id="DynamoVisualProgramming.ZeroTouchLibrary" version="1.2.0"/>
+      <dependency id="Prism" version="4.1"/>
+      <dependency id="NUnit" version="2.6.3"/>
+      <dependency id="DynamoVisualProgramming.ZeroTouchLibrary" version="1.2.0"/>
```

Comment:
How do we update the dependency version attribute, matched with the version of this nuspec?